app/domain/authentication/login_result.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'dry-struct'
+require 'types'
+
+module Authentication  
+  # Response from an authenticator login method
+  class LoginResponse < ::Dry::Struct
```

Comment:
There's a mismatch between the class and file names here. `login_result` vs `LoginResponse`.

Reply:
Thanks, I will fix this.